XRTK.Lumin/Packages/com.xrtk.lumin/package.json

```diff
@@ -13,13 +13,13 @@
     "reality",
     "lumin"
   ],
-  "version": "0.2.1",
+  "version": "0.2.3",
   "unity": "2019.4",
   "license": "MIT",
   "src": "Assets/XRTK.Lumin",
   "author": "XRTK Team (https://github.com/XRTK)",
   "dependencies": {
-    "com.xrtk.core": "0.2.1"
+    "com.xrtk.core": "0.2.3"
```

Comment:
Let's revert the package dependency change for now as there aren't any breaking changes. That way we can get the builds to pass and merge this into development first and do our usual workflow